website_sale_product_attribute_value_filter_existing/controllers/main.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo import http
+from odoo.http import request
+
+from odoo.addons.website_sale.controllers.main import WebsiteSale
+
+
+class ProductAttributeValues(WebsiteSale):
+    def _get_search_domain(
+        self, search, category, attrib_values, search_in_description=True
+    ):
+        # Store used domain in context to be reused after
+        domain = super(ProductAttributeValues, self)._get_search_domain(
```

Comment:
We're on python 3 now! :yum: 

```suggestion
        domain = super()._get_search_domain(
```